Integrations/integration-DemistoRESTAPI.yml

```diff
@@ -0,0 +1,98 @@
+description: Use Demisto server REST APIs
+configuration:
+- display: Demisto Server URL
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Demisto Server API Key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Trust certificate (insecure)
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: User system proxy settings
+  name: proxy
+  defaultvalue: "true"
+  type: 8
+  required: false
+script:
+  script: |
+    var serverURL = params.url;
+    if (serverURL.slice(-1) === '/') {
+        serverURL = serverURL.slice(0,-1);
+    }
+
+    var sendRequest = function(method, url, body) {
+        var requestUrl = serverURL;
+        if (url.slice(-1) != '/') {
+            requestUrl += '/';
+        }
+        requestUrl += url;
+
+        var res = http(
+            requestUrl,
+            {
+                Method: method,
+                Headers: {
+                    'Accept': ['application/json'],
+                    'content-type': ['application/json'],
+                    'authorization': [params.apikey]
+                },
+                Body: body
+            },
+            params.insecure,
+            params.proxy
+            );
+
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Demisto REST APIs - Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+
+        try {
+            var json = JSON.parse(res.Body);
+            return {response:json};
+        } catch (ex) {
+            throw 'Demisto REST APIs - Error parsing response - ' + ex;
+        }
+    };
+
+    switch (command) {
+        case 'test-module':
+            sendRequest('GET','system/config');
+            return 'ok';
+        case 'demisto-api-post':
+            var body = JSON.parse(args.body);
+            return sendRequest('POST',args.url, args.body);
+        case 'demisto-api-get':
+            return sendRequest('GET',args.url);
+
+        default:
+            throw 'Demisto REST APIs - unknown command';
+
+    }
+  type: javascript
+  commands:
+  - name: demisto-api-post
+    arguments:
+    - name: url
+      description: Request URL
+    - name: body
+      description: Body of HTTP POST
+    description: send HTTP POST request
+  - name: demisto-api-get
+    arguments:
+    - name: url
+      description: Request URL
```

Comment:
Should be "Request URI" and not URL.
And you should provide example like: `/users`. So they won't use their full url.